CONTRIBUTORS

```diff
@@ -28,3 +28,4 @@ Utkarsh Anand
 Tobias Klauser
 Tim Tianyang Chen
 Ed Maste
+Sumukha PK
```

Comment:
Please also add yourself to AUTHORS file, see comment on top of CONTRIBUTORS for explanation.

Reply:
Should I remove myself from the CONTRIBUTORS ?

Reply:
No, leave it in both. One is code authors, another is copyright holders. For example, in my case I have my name in CONTRIBUTORS, but Google in AUTHORS.
It's legal requirements, but fortunately we need to go through this only for the first commit.